src/main/kotlin/no/nav/syfo/infrastructure/kafka/ExpiredForhandsvarselProducer.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.infrastructure.kafka
+
+import no.nav.syfo.application.IExpiredForhandsvarselProducer
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.domain.Varsel
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class ExpiredForhandsvarselProducer(private val producer: KafkaProducer<String, Varsel>) :
+    IExpiredForhandsvarselProducer {
+
+    override fun send(personIdent: PersonIdent, varsel: Varsel): Result<Varsel> =
+        try {
+            producer.send(
+                ProducerRecord(
+                    TOPIC,
+                    UUID.randomUUID().toString(),
+                    varsel,
+                )
+            ).get()
+            Result.success(varsel)
+        } catch (e: Exception) {
+            log.error("Exception was thrown when attempting to send expired forhandsvarsel: ${e.message}")
+            Result.failure(e)
+        }
+
+    companion object {
+        private const val TOPIC = "teamsykefravr.arbeidsuforhet-expired-forhandsvarsel"
```

Comment:
Regner med det kommer ny PR på å opprette denne topicen og definere DTO vi skal publisere på Kafka (hvis vi ikke vil publisere hele Varsel-objektet)

Reply:
Riktig, vi må publishe topicet til kubernetes, sånn som: https://github.com/navikt/isbehandlerdialog/blob/master/.nais/kafka/ubesvartmelding.yaml

Reply:
Akkurat opprettelse av topic går vel fint an å legge inn nå. Akkurat hva vi slenger ut på kafka er for meg foreløpig litt uklart 😊

Reply:
Ja, går fint an å vente med begge deler til ny PR, bare så vi ikke glemmer det :+1: